ciongke/Makefile

```diff
@@ -98,4 +104,15 @@ sinker-image:
 sinker-deployment:
 	kubectl apply -f sinker_deployment.yaml
 
-.PHONY: hook-image hook-deployment hook-service test-pr-image sinker-image sinker-deployment
+deck-image:
```

Comment:
It's really just so that I can type `make hook-deployment` rather than `kubectl apply -f hook_deployment.yaml`. Laziness :)